frontend/src/pages/manage-member/AddMember.jsx

```diff
@@ -166,10 +158,10 @@ const AddMember = ({
         }
       >
         <Row gutter={[12, 12]}>
-          <Col span={12}>
+          <Col span={24}>
             <Form.Item
-              name="first_name"
-              label="First Name"
+              name="name"
+              label="Member name"
```

Comment:
This Member name input field was disabled when editing a member. 
I think the admin should be able to update the member/organisation name.